core/src/tpu.rs

```diff
@@ -280,9 +283,11 @@ impl Tpu {
             prioritization_fee_cache,
         );
 
+        let client = ForwardingClientOption::ConnectionCache(connection_cache.clone());
```

Comment:
In this PR unconditionally use ConnectionCache. In one of the next PRs, I will add a new constructor and this `ForwardingClientOption` will be set up in the validator.rs.

Reply:
I suggest you patch the selection right away to avoid dead code (if it is not too much hassle). I think you already have some CLI that enables TPU-client-next no?

Reply:
No, I don' t have cli changes merged yet.